src/App/components/Authentication/index.js

```diff
@@ -0,0 +1,41 @@
+import React, {PropTypes} from 'react'
+import login from 'package/utils/login'
+import logout from 'package/utils/logout'
+import Heading from 'package/components/Heading'
```

Comment:
As for including instructor-specific stuff, Joel and I have discussed this at length and decided it would be good to expand egghead-ui to include components/screens/utils that can be rendered in egghead-rails - so UI development will gradually move into other repos and egghead-rails will eventually contain no UI other than routing, authentication, and endpoints. That is the long term plan. Move UI out into repos like egghead-ui.